### ISSUE TITLE
Sonarr V3 updated broken style

### ISSUE THREAD

**Issue by JourneyOver** [iFelix18/Darkerr#17]
Been using the dev branch of darkerr since I've been using the V3 release of sonarr, and it's been working great and all, but looks like an update (either release `3.0.1.384` or the recent one `3.0.1.388`) hit and has changed some CSS around, as the style is pretty much borked now xD

Just thought I'd let ya know :p

![chrome_03-05-2019_15-48-37](https://user-images.githubusercontent.com/13910285/53839922-5bbb7b80-3f5e-11e9-99e4-f57f85cc529a.jpg)
![03-05-2019_15-51-11](https://user-images.githubusercontent.com/13910285/53839985-83aadf00-3f5e-11e9-958c-c6a7fddc7b25.png)

Dunno what all is broken, but those two above are the most evident for sure.

**Comment by iFelix18** [comment]
Yeah, I noticed. According to [the forum](https://forums.sonarr.tv/t/latest-v3-update-broke-custom-css/) the classes are generated automatically, and it is not excluded that they can change again in the future.

For example
old class  : `.SeriesIndexPoster-title-1LWvk`
new class : `.SeriesIndexPoster-title-263Pq`

To avoid future break I should update my theme entirely with [classes like](https://www.w3schools.com/cssref/sel_attr_begin.asp): `[class^='SeriesIndexPoster-title']`

It will take a bit 'of time...

**Comment by JourneyOver** [comment]
Ah wow, figured that's probably what they where doing but I never really looked much into the CSS of sonarr especially for v3 xD it's pretty much the same thing plex does though as well and the reasoning on why I use the whole `[class^='whatever']` system for it.

**Comment by iFelix18** [comment]
Darkerr 3.0.0-beta8 has been released. It should solve every problem with the new Sonarr updates, I await feedbacks.

**Comment by JourneyOver** [comment]
@iFelix18 	Looks good, everything seems to be back to working order again and nothing seems to be messed up from my quick look through. Also why did you delete your jackett theme or are you moving it to it's own repository?

**Comment by iFelix18** [comment]
> Also why did you delete your jackett theme or are you moving it to it's own repository?

It was more like a test, it was never even complete, and it never left the develop branch. I consider Jackett a "background service", I use very little Jackett's UI once configured in Sonarr and other -rr services.
So I decided to remove the theme for Jackett, and let Darkerr return what it was, a dark theme only for the -rr services.

**Comment by JourneyOver** [comment]
Ahh okay, anyways guess this issue can be closed now (will close it with this comment) since latest dev looks to have taken care of everything so far.

**Comment by JourneyOver** [comment]
Just going to re-open this issue:

Looks like CSS has changed again in version 3.0.2.549, not as broken as before as text is actually readable but it's no longer fully themed.

![chrome_07-28-2019_21-56-22](https://user-images.githubusercontent.com/13910285/62019042-c5912780-b182-11e9-8c9c-d1fa58ae801f.jpg)
![chrome_07-28-2019_21-56-37](https://user-images.githubusercontent.com/13910285/62019043-c5912780-b182-11e9-8969-15d4c9579fb0.png)

**Comment by arogl** [comment]
I'll add some more shots:
http://localhost:8989/calendar
![image](https://user-images.githubusercontent.com/1115472/62033277-f2027d80-b22e-11e9-8c57-fa7d978a890f.png)

Popup box when clicking on a show on the calendar screen
![image](https://user-images.githubusercontent.com/1115472/62033371-2118ef00-b22f-11e9-9c77-202c6fc3d135.png)

Popup box on clicking manual import
![image](https://user-images.githubusercontent.com/1115472/62033430-41e14480-b22f-11e9-955e-85653ff41179.png)

On a show screen
![image](https://user-images.githubusercontent.com/1115472/62033498-6f2df280-b22f-11e9-83f1-b2cd77f8e8f1.png)

Most popups

I'm not sure how often this will happen, but I notice they are considering a default dark css - prompted by @iFelix18

**Comment by JourneyOver** [comment]
> but I notice they are considering a default dark css - prompted by @iFelix18

This would actually be nice, then we wouldn't have to worry about things breaking like this.

**Comment by iFelix18** [comment]
Thanks, I'll take a look in the next few days.

**Comment by iFelix18** [comment]
I took a look. Every single class has changed, it will take a bit of time, it's a long and boring job... it's getting hard to keep this theme with all these updates.

**Comment by JourneyOver** [comment]
Pretty much how it is with me and Plex as well xD They make it hard to keep a theme going with the updates that they pull out of their ass that ends up changing so much sometimes :/

**Comment by arogl** [comment]
I have installed [dark reader](https://darkreader.org/), out of the box it does a pretty good job of having a dark style for Sonarr.
